### DISCUSSION TITLE
What is the purpose of the "Add Citation Links..." in the "Better BibTeX" context menu in Zotero?

### DISCUSSION THREAD

**Discussion by ScientificProgrammer** [retorquere/zotero-better-bibtex#2236] [Q&A] [answered]
Hi,

Can anyone tell me the purpose of the `Add citation links...` item in the _Better BibTeX_ add-on's context menu in Zotero (please see screenshot below)?

![Zotero-Screenshot_2022-08-07_123544_BetterBibtex_AddCitations](https://user-images.githubusercontent.com/4348294/183304054-6352d6f5-2cc9-4a5c-9f18-53b32c4655b3.png)

I'm just curious what it does. Whenever I've tried using it, the standard dialog box that I normally use for inserting Zotero references into a target document appears. However, even if I select some references, I don't notice any change in either the source or target citation.

To find the answer to my question, I've tried searching the [documentation](https://retorque.re/zotero-better-bibtex/) for _Better BibTeX_, [the Zotero Forums](https://forums.zotero.org/discussions), and the web. 

In searching the web, the only result I could find was for a [single line of code](https://github.com/retorquere/zotero-better-bibtex/blob/64a3a59261b4d4d95da9dd027764e513c98d9ef2/locale/en-US/zotero-better-bibtex.dtd#L232) with the relevant string in the `zotero-better-bibtex.dtd` file for the `en-US` locale here in the project's Github repo.

Thanks for making this great add-on!

**Comment by retorquere** [comment, marked as answer]
It's for the Citation Graph translator -- if you write `cites: citationkey1, citationkey2, ...` in the `extra` field, the CG translator will export a GML file showing how literature hangs together. The `add citation links` will pop up a picker and will write the picked items into the extra field that way. It was useful to me for citation analysis for me at some point long ago.

**Reply by ScientificProgrammer** [reply]
That's an incredibly useful feature. Thank you for the prompt reply, as well as for making this great tool available to the world!
